src/SimpleGraphs/simplegraph.jl

```diff
@@ -21,6 +21,26 @@ end
 
 eltype(x::SimpleGraph{T}) where {T} = T
 
+# SimpleGraph(g)
+"""
+    SimpleGraph{T}(g::AbstractGraph)
+
+Construct a `SimpleGraph{T}` from an `AbstractGraph`. 
+If `g` is a `SimpleDiGraph`, then every directed edge in `g` is 
+added as an undirected edge.
+"""
+SimpleGraph(g::AbstractGraph) = SimpleGraph{Int}(g)
```

Comment:
```suggestion
SimpleGraph(g::AbstractGraph) = SimpleGraph{eltype(g)}(g)
```
or 
```suggestion
SimpleGraph(g::AbstractGraph{T}) where {T} = SimpleGraph{T}(g)
```
so in that case, the constructed graph takes the eltype from the passed in graph. This is more consistent with existing constructors such as
```julia
SimpleGraph(g::SimpleGraph{T}) where {T} = SimpleGraph{T}(g)
```